packages/webapp/src/lib/services/RegistryManager.ts

```
@@ -0,0 +1,64 @@
+		try {
+			return new URL(window.location.href).searchParams.has('registry');
+		} catch (error) {
+			throw new Error('Failed to check if registry parameter exists: ' + (error instanceof Error ? error.message : String(error)));
+		}
+	}
+
+	static getRegistryParam(): string | null {
+		try {
+			return new URL(window.location.href).searchParams.get('registry');
+		} catch (error) {
+			throw new Error('Failed to get registry parameter: ' + (error instanceof Error ? error.message : String(error)));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)